include/IMeshBuffer.h

```diff
@@ -649,6 +674,14 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
             return ((uint8_t*)mappedAttrBuf->getPointer())+ix;
         }
 
+		//! Accesses given index of given attribute. Index number is incremented by `baseVertex`.
+		/** If component count of given attribute is less than 4, only first ones of output vector's members will be written.
```

Comment:
Big note for this function, NOT ALL FORMAT CONVERSIONS TO RGBA32F/XYZW32F ARE IMPLEMENTED!